lang/es.xml

```diff
@@ -3,29 +3,45 @@
 <lang>
 	<main_menubar>
 		<file name="Archivo">
-			<open_pref_path>Abrir la Carpeta Pref</open_pref_path>
-			<install_firmware>Instalar Firmware</install_firmware>
-			<install_pkg>Instalar un .pkg</install_pkg>
-			<install_zip>Instalar un .zip, .vpk</install_zip>
+			<open_pref_path>Abrir la carpeta de preferencias</open_pref_path>
```

Comment:
The pref path isn't the settings folder of the emulator, but the folder where the emulated PS Vita filesystem is located. So this translation would be closer to _Abrir la carpeta del sistema de archivos de PS Vita emulado_ or _Abrir la carpeta del sistema de archivos de PS Vita_.

Reply:
in french i have juste translate pref path by Dossier Pref lol Pref like Preferé, is just short word